ts/modules/price-indicator.src.ts

```diff
@@ -169,6 +169,14 @@ addEvent(H.Series, 'afterRender', function (): void {
 
             yAxis.cross = serie.lastVisiblePrice;
             lastPoint = points[pLength - crop];
+
+            if (serie.crossLabel) {
+                serie.crossLabel.destroy();
+                // Set to undefined to avoid collision with
+                // the yAxis crosshair #11480
+                delete yAxis.crossLabel;
```

Comment:
The `destroy` function returns undefined, so we can do this shorthand: `serie.crossLabel = serie.crossLabel.destroy();`